app/models/spree/order_updater_decorator.rb

```diff
@@ -65,8 +65,16 @@ def canceled_and_paid_for?
       order.canceled? && paid?
     end
 
+    def canceled_and_not_paied_for?
```

Comment:
I think this is supposed to be: `canceled_and_not_paid_for?`

Reply:
lol yes :see_no_evil: